src/lib/utils/sorter.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Sort comparator function
+ * Sort numbers, strings and dates properly in ascending order
+ *
+ * @param {Object} a - First object to compare
+ * @param {Object} b - Second object to compare
+ * @returns {Number} -1, 0, or 1
+ */
+export default function sorter (a, b) {
```

Comment:
Not necessary, but you could also use `d3.ascending` from [d3-array](https://d3js.org/d3-array/sort#ascending) (since we already have a dependency on it.

Reply:
Yknow, that's a much better solution and what I should have done to begin with.